### PR TITLE
feat: add users

### LINKED ISSUE #321: Add users
### Describe the feature

The ory-svc should receive creations and updates for users from Ory and emits them as events into our services. Our user-svc should react accordingly to them.

### Why is this feature needed?

We need to duplicate some of the users information into our user-svc to keep track of some basic information.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by MaxSchaefer**
## Which issues does this pull request close?
closes #321

### CODE REVIEW

File: dev/docker-compose.yaml [hunk 1]
```diff
@@ -31,8 +37,10 @@ services:
       - full
     restart: unless-stopped
     volumes:
-      - ../apisix/apisix.yaml:/usr/local/apisix/conf/apisix.yaml:ro
-      - ./dapr.yaml:/usr/local/apisix/dapr.yaml:ro
+      - ./dapr.yaml:/usr/local/apisix/dapr/config.yaml:ro
+      - ../apisix/apisix.yaml:/usr/local/apisix/conf/apisix.yaml:ro # for hot-reloading
+    environment:
+      ORY_WEBHOOK_BASIC_AUTH: '{"username":"ory","password":"2341"}'
```
Comment (FoseFx) [inline dev/docker-compose.yaml:43]: we should probably document all values in this file in a README for this directory

File: apisix/apisix.yaml [hunk 2]
```diff
@@ -30,6 +42,19 @@ services:
...
+    upstream_id: dapr-http
+    plugins:
+      cors:
+        allow_origins: "**"
+        allow_methods: "**"
+        allow_headers: "**"
+        expose_headers: "**"
+        max_age: 5
+        allow_credential: true
+        _meta:
```
Comment (FoseFx) [inline apisix/apisix.yaml:55]: what does this do? please add a comment

File: apisix/apisix.yaml [hunk 3]
```diff
@@ -30,6 +42,19 @@ services:
         headers:
           add:
             dapr-app-id: user-svc
+  - id: ory-svc
+    upstream_id: dapr-http
+    plugins:
+      cors:
+        allow_origins: "**"
```
[the remark this answers is not in the document]
Comment (MaxSchaefer) [reply apisix/apisix.yaml:49]: https://github.com/helpwave/services/issues/342

File: Dockerfile.standalone [hunk 4]
```diff
@@ -76,6 +77,7 @@ RUN chmod +x \
 	run-migrations.sh
 
 ENV DAPR_APP_ID=$SERVICE
+ENV LOG_LEVEL=$LOG_LEVEL
```
[the remark this answers is not in the document]
Comment (MaxSchaefer) [reply Dockerfile.standalone:80]: Good catch. Didn't have in mind that we already had a `LOV_LEVEL` env.

**Review by FoseFx — APPROVED** [review]

File: dev/docker-compose.yaml (hunk 1 above)
Comment (MaxSchaefer) [reply dev/docker-compose.yaml:43]: I've added a `.env.example` into the `apisx/` directory. The other `SECRETSTORE_` envs are documented inside the `README.md`.
Yes, we should document our current Dapr and APISIX workflows/setup when the config/setup is no longer so volatile.

**Pull request merged**